pkg/scheduler/algorithm/priorities/node_label_test.go

```diff
@@ -47,7 +47,7 @@ func TestNewNodeLabelPriority(t *testing.T) {
 			expectedList: []schedulerapi.HostPriority{{Host: "machine1", Score: 0}, {Host: "machine2", Score: 0}, {Host: "machine3", Score: 0}},
 			label:        "baz",
 			presence:     true,
-			test:         "no match found, presence true",
+			name:         "no match found, presence true",
```

Comment:
on the subject of cleaning up subtest names, replacing ', ' with '/' in these tests would be nice.